tooling/Slack-channels.md

```diff
@@ -0,0 +1,17 @@
+##Slack
+
+##Join slack: broadinstitute.slack.com
+
+Slack is a major communication method for the team for questions and discussion. Saturn-team is the primary team channel.
```

Comment:
I think some sort of clarifiction about #saturn vs #saturn-team would be good here. Something like:

"#saturn-team is the private team channel for team discussion and alerting other members about absences etc. while #Saturn is a public channel where other teams can bring questions to the Saturn team."

Reply:
I added descriptions to the #saturn and #saturn-team channels, as well as a few others. Some of them I wasn't quite clear enough on the details to annotate (e.g. the nuances of #dsp-engineering, and #dsp-workbench, and the channel descriptions aren't very helpful). I put a TODO on it, but feel free to put suggestions here or in slack and I can drop them in if you want.